as_pacmod.dbc

```diff
@@ -771,6 +775,36 @@ BO_ 1536 WATCHDOG_RPT: 8 PACMOD
  SG_ MOD_SYSTEM_PRESENT_FAULT : 61|1@0+ (1,0) [0|1] ""  CUSTOMER_ECU
  SG_ MINI_SYSTEM_PRESENT_FAULT : 62|1@0+ (1,0) [0|1] ""  CUSTOMER_ECU
 
+BO_ 316 BRAKE_DECCEL_CMD: 8 CUSTOMER_ECU
+ SG_ ENABLE : 0|1@0+ (1,0) [0|1] ""  PACMOD
+ SG_ IGNORE_OVERRIDES : 1|1@0+ (1,0) [0|1] ""  PACMOD
+ SG_ CLEAR_OVERRIDE : 2|1@0+ (1,0) [0|1] ""  PACMOD
+ SG_ BRAKE_DECCEL_CMD : 15|16@0+ (0.001,0) [0|10] "m/s^2"  PACMOD
+ SG_ XBR_EBI_MODE : 25|2@0+ (1,0) [0|2] ""  PACMOD
+ SG_ XBR_PRIORITY : 27|2@0+ (1,0) [0|3] ""  PACMOD
+ SG_ XBR_CONTROL_MODE : 29|2@0+ (1,0) [0|2] ""  PACMOD
```

Comment:
I thought the final decision was to split up the DECCEL_CMD and the XBR bits into a _CMD and an _AUX_CMD? Did I misunderstand the outcome?